packages/curriculum-compiler-json/__tests__/fixtures/exercise/sample/ast.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "root",
+  "children": [
+    {
+      "type": "yaml",
+      "value":
```

Comment:
why is the value empty? is this valid json?

Reply:
That's valid... For some reason the `value`, the plaintext of the yaml field, is on the next line. Looks like something a linter would cause...